build/helpers/webpack.js

```diff
@@ -155,3 +155,43 @@ function mergeStats(stats) {
 }
 
 exports.mergeStats = mergeStats;
+
+/**
+ * Extract tmp folder hash from Webpack stats file
+ *
+ * @param {!Object} stats
+ * @returns {!Object}
+ */
+const getHashFromStats = (json) => {
```

Comment:
```suggestion
const getTmpHashFromStats = (json) => {
```

+ Раз это хэлпер (не экспортируется), то сделай его через function declaration и опусти в самый низ файла

Reply:
Параметр должен быть stats, а не json